x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -239,10 +239,8 @@ const TimelineDescriptionComponent: React.FC<FlyoutHeaderProps> = ({ timelineId
   );
   return (
     <EuiText size="s" data-test-subj="timeline-description">
-      {description.length ? (
-        <LineClamp key={description.length} lineClampHeight={4.5}>
-          {description}
-        </LineClamp>
+      {description ? (
```

Comment:
I think that I introduced this bug.  😥

This type is misleading. `getTimeline` returns a `TimelineModel` and `description` is a mandatory field. If typing is right, it should never be undefined.

Reply:
Yup, I noticed that. This PR is just a patch to prevent the crash. 
I am not sure if it's correct to change _title_ and _description_ to optional in the `TimelineModel`. Or, on the other hand, these `undefined` values shouldn't happen, and we have to go deeper and find the root problem.
Do you know that? I wanted to talk about that with @XavierM next week when he comes back.